src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -148,10 +158,35 @@ class ReportActionContextMenu extends React.Component {
         return lodashGet(message, 'text', '');
     }
 
+    /**
+     * Can the current user edit this report action?
+     *
+     * @return {Boolean}
+     */
+    canEdit() {
+        return this.props.reportAction.actorEmail === this.props.session.email
+            && this.props.reportAction.reportActionID;
+    }
+
+    /**
+     Confirms the deletion of the comment and hides the confirmation modal.
```

Comment:
Don't need this method doc (or the one below) at all (like PHP, they are only needed if the method is non-obvious)